charts/rancher-monitoring/v0.1.4/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.9-rc1
```

Comment:
We _do_ apparently want this to be available in 2.4.9, previously only 0.1.2 was available there.